tests/test_validator.py

```diff
@@ -31,15 +31,15 @@ def test_validate_server():
     assert validate_server("http://opendata.cern.ch") is True
     assert validate_server("http://opendata-dev.cern.ch") is True
     assert validate_server("http://0.0.0.0:5000") is True
-    pytest.raises(click.BadParameter, validate_server, "https://opendata.cern.ch")
-    pytest.raises(click.BadParameter, validate_server, "opendata.cern.ch")
+    pytest.raises(SystemExit, validate_server, "https://opendata.cern.ch")
+    pytest.raises(SystemExit, validate_server, "opendata.cern.ch")
 
 
 def test_validate_range():
     """Test validate_range()."""
     assert validate_range(range="3-9", count=10) is True
     assert validate_range(range="1-5", count=8) is True
-    pytest.raises(click.BadParameter, validate_range, "0-9", 5)
-    pytest.raises(click.BadParameter, validate_range, "0", 5)
-    pytest.raises(click.BadParameter, validate_range, "1-9", 5)
-    pytest.raises(click.BadParameter, validate_range, "3-2", 5)
+    pytest.raises(SystemExit, validate_range, "0-9", 5)
```

Comment:
Following up the change from `click.BadParameter` to `SystemExit`, can you please amend `validator.py`'s docstrings?

Reply:
Done 👍